darts/models/torch_forecasting_model.py

```diff
@@ -746,16 +779,15 @@ def _load_untrained_model(self, folder):
             model = torch.load(f)
         return model
 
-    def _prepare_tensorboard_writer(self, continue_training: bool = False):
+    def _prepare_tensorboard_writer(self):
         runs_folder = _get_runs_folder(self.work_dir, self.model_name)
         if self.log_tensorboard:
-            if not continue_training:
-                shutil.rmtree(runs_folder, ignore_errors=True)
+            if self.total_epochs == 0:
+                tb_writer = SummaryWriter(runs_folder, purge_step=self.total_epochs)
+            else:
```

Comment:
The logic in this else block was executed in the case where `continue_training == False` before, which corresponded to the case where `epochs==0`, meaning we were training the model from scratch. Now it is executed when `self.total_epochs > 0`. Now it is executed when `self.total_epochs > 0`, which is the opposite case, no? It seems to me we are executing the same code in a different case, or am I missing something? Or, if I'm right, why is that the case?

Reply:
Good catch! You're absolutely right.

Reply:
@mkos you still have to do this one right?